mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java

```diff
@@ -117,4 +120,15 @@ public void showMoreHistory(int existingItemCount) {
             handleTransactionsHistory(existingItemCount);
         }
     }
+
+    @Override
+    public void fetchTransactions() {
```

Comment:
@Grandolf49 remove this empty function.

Reply:
Hi @rachittt, Thanks for pointing out. I'll work that out.